### DISCUSSION TITLE
Bootstrap variables don't work in storybook

### DISCUSSION THREAD

**Discussion by NavroO** [storybookjs/storybook#20104] [Help]
Hi 
I am trying to install a storybook for my Next application. The installation of Next-Image, font-awsome and the bootstrap itself was successful. However, I encountered another problem. For some reason storybook is not loading my scss variables which overwrite the default ones from bootstrap. By in storybook I have the default bootstrap colors. I tried installing post-css, importing my theme into preview.js file however, these did not work. Storybook either throws node_module error or just any kind of installation has no effect. How can I remedy this?

preview.js

https://pastebin.com/CLHUMR3m

main.js

https://pastebin.com/WLZ7wHpg

preview-head.html

https://pastebin.com/5ELxmiKD

The bootstrap itself works fine the problem is with custom css varibles. My varibles file is in the theme folder

<img width="244" alt="image" src="https://user-images.githubusercontent.com/42099711/205892354-5840f1df-3045-4701-9c16-e48726729e50.png">

**Comment by dibs** [comment]
Did you figure this out. I'm having issues with the same problem at the moment
